lib/posix-process/Makefile.uk

```diff
@@ -16,6 +16,22 @@ LIBPOSIX_PROCESS_SRCS-y += $(LIBPOSIX_PROCESS_BASE)/signals.c
 LIBPOSIX_PROCESS_SRCS-$(CONFIG_LIBPOSIX_PROCESS_CLONE) += $(LIBPOSIX_PROCESS_BASE)/clone.c
 LIBPOSIX_PROCESS_SRCS-$(CONFIG_LIBPOSIX_PROCESS_CLONE) += $(LIBPOSIX_PROCESS_BASE)/clonetab.ld
 
+ifneq ($(filter y,$(CONFIG_LIBPOSIX_PROCESS_TEST) $(CONFIG_LIBUKTEST_ALL)),)
+	LIBPOSIX_PROCESS_SRCS-y += $(LIBPOSIX_PROCESS_BASE)/tests/test_tls.c
+
+	ifeq (x86_64,$(CONFIG_UK_ARCH))
+		LIBPOSIX_PROCESS_SRCS-y += $(LIBPOSIX_PROCESS_BASE)/tests/test_syscall_shim_helper.s
+		LIBPOSIX_PROCESS_SRCS-y += $(LIBPOSIX_PROCESS_BASE)/tests/test_syscall_shim.c
+		UK_PROVIDED_SYSCALLS-$(CONFIG_LIBPOSIX_PROCESS) += fork-0
```

Comment:
Although this is guarded, I would still add a comment saying that this is not a real fork